.github/workflows/test.yaml

```diff
@@ -74,6 +74,6 @@ jobs:
           whereis protoc-gen-mypy
           whereis protoc-gen-mypy_grpc
           echo $GITHUB_PATH
-        uses: "bufbuild/buf-setup-action@v1.30.1"
+      - uses: "bufbuild/buf-setup-action@v1.30.1"
```

Comment:
This was previously broken and the test file wasn't running at all 😱 Fixing it here so it runs